CONTRIBUTING.md

```diff
@@ -9,6 +9,47 @@
 
 ## Setting up development environment
 
+### Preparing the development environment (Windows)
+
+Several packages & environment variables are necessary to build in a Windows
+environment:
+
+1. Download & install the latest [Go MSI][go-msi].
+
+1. Download & install [Make for Windows][make-for-windows].
+
+1. Open `cmd` and add make to the user PATH:
+
+  ```bat
+  setx path "%PATH%;C:\Program Files (x86)\GnuWin32\bin"
+  ```
+
+1. (Optional) Install [Windows Terminal][windows-terminal].
```

Comment:
I don't think I would bother the user with installing the Windows Terminal. I mean, it's a good app that I use and would recommend to anyone, but it's orthogonal to the instructions on building the binary and it's another step, even if optional, which you could argue is even worse, because the user now needs to make a decision, which is the harder to make the more uninformed the user is. 🙂 

I'd rather keep the steps as minimal as possible.